VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+ 
+### What are software requirements?
+
+In addition to running a Cosmos Hub node, validators should develop monitoring, alerting and management solutions.
+
+### What are bandwidth requirements?
+
+The Cosmos network has the capacity for very high throughput relative to chains like Ethereum or Bitcoin.
+We recommend that the data center nodes only connect to trusted full-nodes in the cloud or other validators that know each other socially. This relieves the data center node from the burden of mitigating denial-of-service attacks.
+Ultimately, as the network becomes more heavily used, multigigabyte per day bandwidth is very realistic.
+
+### What does running a validator imply in terms of logistics? 
+
+A successful validator operation will require the efforts of multiple highly skilled individuals and continuous operational attention. This will be considerably more involved than running a bitcoin miner for instance.
+
+### How to handle key management?
+
+Validators should expect to run an HSM that supports ed25519 keys. Here are the current options:
+ * YubiHSM 2
+ * Ledger Nano S
+ * Potential options
+ * Ledger BOLOS SGX enclave
+ * Thales nShield support
+ * Tendermint SGX enclave
+
+### What can validators expect in terms of operations? 
+
+Running effective operation is the key to avoiding unexpectedly unbonding or being slashed. This includes being able to respond to attacks, outages, as well as to maintain security and isolation in your data center.
+
+### What are the maintenance requirements?
+
+Validators should expect to perform regular software updates to accommodate upgrades and bug fixes. There will inevitably be issues with the network early in its bootstrapping phase that will require substantial vigilance. 
+
+### How can validators protect themselves from denial-of-service attacks?
+ 
+Denial-of-service attacks occur when an attacker sends a flood of internet traffic to an IP address to prevent the server at the IP address from connecting to the internet.
+An attacker scans the network, tries to learn the IP address of various validator nodes and disconnect them from communication by flooding them with traffic. 
+
+One recommended way to mitigate these risks is for validators to carefully structure their network topology in a so-called sentry node architecture. 
+Validator nodes should only connect to full-nodes they trust because they operate them themselves or are run by other validators they know socially. A validator node will typically run in a data center. Most data centers provide direct links the networks of major cloud providers. The validator can use those links to connect to sentry nodes in the cloud. 
+
+Sentry nodes can be quickly spun up or change their IP addresses. Because the links to the sentry nodes are in private IP space, an internet based attacked cannot disturb them directly. This will ensure validator block proposals and votes always make it to the rest of the network.
```

Comment:
add something like "this shifts the burden of DoS from the validator node directly to it's sentry nodes, and may require new sentry nodes be spun up or activated to mitigate attacks on existing ones"

Reply:
Addressed